tools/render-test/render-d3d11.cpp

```diff
@@ -80,10 +74,10 @@ class D3D11Renderer : public Renderer, public ShaderCompiler
     {
         ShaderInputType type;
         InputBufferType bufferType;                        // Only valid if `type` is `Buffer`
-        ID3D11ShaderResourceView * srv = nullptr;
```

Comment:
This cleanup I fully support. :)

Unstated policy in the Slang codebase is that pointer variables are declared as `T* v;` and not `T *v;` or `T * v;` or any other variation. In general we shouldn't be declaring multiple variables at once, so the supposed gotchas won't affect us.